pkg/rpcclient/wsclient_test.go

```diff
@@ -209,26 +224,37 @@ func TestWSClientEvents(t *testing.T) {
 	wsc.receivers[chan<- *state.AppExecResult(aerCh2)] = []string{"5"}
 	wsc.subscriptions["6"] = &executionReceiver{filter: &neorpc.ExecutionFilter{State: &fault}, ch: aerCh3}
 	wsc.receivers[chan<- *state.AppExecResult(aerCh3)] = []string{"6"}
+
+	added := mempoolevent.TransactionAdded
+	removed := mempoolevent.TransactionRemoved
+	wsc.subscriptions["7"] = &mempoolEventReceiver{ch: memChAll}
+	wsc.receivers[chan<- *result.MempoolEvent(memChAll)] = []string{"7"}
+	wsc.subscriptions["8"] = &mempoolEventReceiver{filter: &neorpc.MempoolEventFilter{Type: &added}, ch: memChAdded}
+	wsc.receivers[chan<- *result.MempoolEvent(memChAdded)] = []string{"8"}
+	wsc.subscriptions["9"] = &mempoolEventReceiver{filter: &neorpc.MempoolEventFilter{Type: &removed}, ch: memChRemoved}
+	wsc.receivers[chan<- *result.MempoolEvent(memChRemoved)] = []string{"9"}
 	// MissedEvent must close the channels above.
 
 	wsc.subscriptionsLock.Unlock()
 	close(startSending)
 
 	var (
-		b1Cnt, b2Cnt                                      int
-		aer1Cnt, aer2Cnt, aer3Cnt                         int
-		ntfCnt                                            int
-		expectedb1Cnt, expectedb2Cnt                      = 1, 1    // single Block event
-		expectedaer1Cnt, expectedaer2Cnt, expectedaer3Cnt = 2, 2, 0 // two HALTED AERs
-		expectedntfCnt                                    = 1       // single notification event
-		aer                                               *state.AppExecResult
+		b1Cnt, b2Cnt                                            int
+		aer1Cnt, aer2Cnt, aer3Cnt                               int
+		ntfCnt                                                  int
+		memAllCnt, memAddCnt, memRemCnt                         int
```

Comment:
s/memAddCnt/memAddedCnt
s/memRemCnt/memRemovedCnt

To match memChAdded and memChRemoved.